baselines/fjord/fjord/conf/config.yaml

```diff
@@ -0,0 +1,8 @@
+---
+hydra:
+  run:
+    dir: ./runs/${now:%Y-%m-%d}:${now:%H-%M-%S}
```

Comment:
I wonder if we need this. Hydra by defaults creates an experiment for your directory following the structure: `outputs/date/time`. E.g. `outputs/Y-M-D/h-m-s/`

Reply:
I would prefer the flat structure run directories that is alphanumerically sorted by the filesystem.